src/misc.jl

```diff
@@ -4,10 +4,10 @@
 #
 # See: https://en.wikipedia.org/wiki/Multivariate_gamma_function
 #
-function logmvgamma(p::Int, a::Float64)
-    res::Float64 = p * (p - 1.0) / 4.0 * log(pi)
+function logmvgamma(p::Int, a::Real)
+    res = p * (p - 1) / 4 * log(pi * one(a))
     for ii in 1:p
-        res += lgamma(a + (1.0 - ii) / 2.0)
+        res += lgamma(a + (1 - ii) / 2)
```

Comment:
...but this one is tricky. `(1 - ii)` would have to be converted to `typeof(a)` before the division.